core/admin/start.py

```diff
@@ -46,9 +46,14 @@ def test_DNS():
 
 test_DNS()
 
+bind_addr = ":80"
```

Comment:
Force push (for including merged ##2290) got rid of a comment here :frowning: 

Could be made into
```python
# If SUBNET6 is defined, gunicorn must listen on IPv6 as well as IPv4
bind_addr = "[::]:80" if os.environ.get("SUBNET6") else ":80"
```

I have decided against this for readability though, but could implement this short version if this is wished/according to code-style-standards.